DBTableToHBaseTable/README.rst

```diff
@@ -0,0 +1,85 @@
+=============================================================
+Batch Database Table To CDAP HBase Table Adapter Configuration
+==============================================================
+
+The ETL Batch Template can be used to create an Adapter that reads from a Batch Source and persists it to a Sink.
+In this example, we will read an entire DB table in batch and use a TableSink to write the database table's rows to HBase.
+
+The config.json contains a sample Adapter configuration that you can use to accomplish the above task. Our sample Adapter uses these components:
+
+- ETLBatch Application Template, since we want to perform ETL in batch
+- Database source, to read data from the database table 
+- Table sink, to write the Kafka events to a Stream
+- A jar file containing the JDBC driver for your database. Along with this, you also need a JSON file that describes the JDBC driver as an external plugin. This file should have the same name as the jar file (with only the extension changed to '.json'). See `mysql-connector-java-5.1.35.json`` and ``postgresql-9.4.json`` as examples.
+
```

Comment:
The formatting is screwed up on this file, as you can see because of the change of color of the text. It doesn't render as rst as you can see when you change the view. It might fix it self if you make the other changes shown here.